Makefile

```diff
@@ -48,3 +48,7 @@ megalint:  ## Run the mega-linter.
 		-v /var/run/docker.sock:/var/run/docker.sock:rw \
 		-v $(shell pwd):/tmp/lint:rw \
 		oxsecurity/megalinter:v7
+
+.PHONY: run
+run:
+	poetry run uvicorn src.main:app --reload
```

Comment:
Was just thinking, the default fastapi port is `8000`, so we might want to switch to another so it doesn't clash with other ports? Could just add something arbitrary like `--port 5010` to the end for now?

Reply:
Good point, just set the port value